src/ORM/Queries/SQLSelect.php

```diff
@@ -249,7 +249,7 @@ public function setLimit($limit, $offset = 0)
             throw new InvalidArgumentException("SQLSelect::setLimit() only takes positive values");
         }
 
-        if (is_numeric($limit) && ($limit || $offset)) {
```

Comment:
The second part of this IF statement would fail if `0,0` was provided. This would effectively prevent you from setting a zero limit.

Reply:
`setLimit()` actually accept strings and array. This seems a bit weird but refactoring this is probably beyond the scope of this card.